doc/api/os.md

```diff
@@ -1170,6 +1170,43 @@ The following error codes are specific to the Windows operating system:
   </tr>
 </table>
 
+### dlopen Constants
+
+If available on the operating system, the following constants
+are exported in `os.constants.dlopen`. See dlopen(3) for detailed
```

Comment:
Care to link http://man7.org/linux/man-pages/man3/dlopen.3.html?

Reply:
Isn't it done automagically. I think we already discussed this.

Reply:
My bad. Yes, tools/doc/html.js will take care of this.